e2e/specs/film.spec.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { test, expect } from '@playwright/test';
+import { mockRoute } from '../apiMock';
+import { zendeskMock } from '../mockResponses';
+
+test.beforeEach(async ({ page }) => {
+  const filmFrontpage = mockRoute({
+    page,
+    path: '**/frontpage-api/v1/filmfrontpage/',
+    fixture: 'film_frontpage',
+  });
+
+  const allMovies = mockRoute({
+    page,
+    path: '**/search-api/v1/search/*',
+    fixture: 'film_all_movies',
+  });
+
+  const zendeskToken = mockRoute({
+    page,
+    path: '**/get_zendesk_token',
+    fixture: 'film_zendesk_token',
+    overrideValue: JSON.stringify(zendeskMock),
+  });
+
+  await page.goto('/film');
+
+  await Promise.all([filmFrontpage, allMovies, zendeskToken]);
+});
+
+test('Can add a movie to the slideshow', async ({ page }) => {
+  await page.getByTestId('dropdownInput').first().fill('Brukerstøtte');
+  await page.getByTestId('dropdown-items').locator('[id="downshift-1-item-0"]').click();
+  await page.getByTestId('elementListItem').getByRole('img').first().click();
+});
+
+test('Can remove movie from list', async ({ page }) => {
+  await page.unroute('**/frontpage-api/v1/filmfrontpage');
+  await mockRoute({
+    page,
+    path: '**/frontpage-api/v1/filmfrontpage/',
+    fixture: 'film_frontpage_update',
+  });
+  await expect(
+    page.getByTestId('elementListItem').filter({ hasText: 'Brukerstøtte' }),
+  ).toBeVisible();
+  await page
+    .getByTestId('elementListItem')
+    .filter({ hasText: 'Brukerstøtte' })
+    .getByTestId('elementListItemDeleteButton')
+    .click();
+  await expect(page.getByTestId('elementListItem').filter({ hasText: 'Brukerstøtte' })).toBeVisible(
+    {
+      visible: false,
+    },
+  );
```

Comment:
Har vi ikke en `not`-ekvivalent av visible?

Reply:
Det finnes ikke!